modules/Microsoft.Compute/galleries/.test/common/deploy.test.bicep

```diff
@@ -46,15 +46,6 @@ module testDeployment '../../deploy.bicep' = {
     enableDefaultTelemetry: enableDefaultTelemetry
     name: '<<namePrefix>>${serviceShort}001'
     lock: 'CanNotDelete'
-    roleAssignments: [
```

Comment:
Hey @shawntmeyer, did the RBAC test fail for you, or how come this was removed?

Reply:
I moved the RBAC assignment for the Resource Group to the dependencies.bicep because the contributor role is needed at the RG level not the subscription level.

Reply:
Was moved to the bottom.